templates/docs/rhpam710-authoring-ha.adoc

```diff
@@ -342,9 +342,7 @@ tcpSocket on port 3306
 |`HTTPS_KEYSTORE` | The name of the keystore file within the secret for Business Central. | `${BUSINESS_CENTRAL_HTTPS_KEYSTORE}`
 |`HTTPS_NAME` | The name associated with the server certificate for Business Central. | `${BUSINESS_CENTRAL_HTTPS_NAME}`
 |`HTTPS_PASSWORD` | The password for the keystore and certificate for Business Central. | `${BUSINESS_CENTRAL_HTTPS_PASSWORD}`
-|`JGROUPS_PING_PROTOCOL` | -- | openshift.DNS_PING
-|`OPENSHIFT_DNS_PING_SERVICE_NAME` | -- | `${APPLICATION_NAME}-rhpamcentr-ping`
-|`OPENSHIFT_DNS_PING_SERVICE_PORT` | -- | 8888
+|`JGROUPS_PING_PROTOCOL` | -- | kubernetes.KUBE_PING
```

Comment:
no need to update adocs, it is auto generated before releases.

Reply:
ok